app/models/manageiq/providers/vmware/infra_manager/vm/remote_console.rb

```diff
@@ -43,7 +43,7 @@ def remote_console_acquire_ticket_queue(protocol, userid)
 
   def remote_console_vmrc_acquire_ticket(_userid = nil, _originating_server = nil)
     validate_remote_console_acquire_ticket("vmrc")
-    ext_management_system.remote_console_vmrc_acquire_ticket
+    ext_management_system.remote_console_vmrc_acquire_ticket(self)
```

Comment:
How about we build the URL here so we don't need to pass `self` into that method?